apps/docs/app/not-found/page.tsx

```diff
@@ -15,25 +15,34 @@ const metadata: Metadata = {
 }
 
 const NotFoundPage = async ({ searchParams: { page } }: { searchParams: { page?: string } }) => {
-  const recommendations = page ? await getRecommendations(page) : null
-
-  return <NotFound recommendations={recommendations} />
-}
-
-const getRecommendations = async (page: string) => {
-  const query = decodeURIComponent(page.replace(/^\/(?:guides|reference)\//, '')).replace(
-    /[_\/-]/g,
-    ' '
-  )
-  if (!query) return null
-
-  const supabase = createClient<Database>(
-    process.env.NEXT_PUBLIC_SUPABASE_URL!,
-    process.env.NEXT_PUBLIC_SUPABASE_ANON_KEY!
+  return (
+    <article className="prose max-w-[80ch]">
+      <h1>We couldn&apos;t find that page</h1>
```

Comment:
Should we keep a mention to `404` in the title?
Like `404: page not found`, to maintain familiarity with the 404 pages pattern @charislam